src/classes/Passage.ts

```diff
@@ -164,40 +171,49 @@ export default class Passage {
      * Create a Magic Link for your app.
      *
      * @param {MagicLinkRequest} magicLinkReq options for creating a MagicLink.
-     * @return {Promise<MagicLinkObject>} Passage MagicLink object
+     * @return {Promise<MagicLink>} Passage MagicLink object
      */
-    async createMagicLink(magicLinkReq: MagicLinkRequest): Promise<MagicLinkObject> {
+    async createMagicLink(magicLinkReq: CreateMagicLinkRequest): Promise<MagicLink> {
         try {
-            const response = await fetch({
-                body: magicLinkReq,
+            const configuration = new Configuration({
```

Comment:
all these configuration objects needs the `'Passage-Version': config.version` header still

Reply:
Added.